packages/ceramic-core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -160,7 +160,7 @@ export default class EthereumAnchorService extends AnchorService {
      * @param anchorProof - Anchor proof instance
      */
     async validateChainInclusion(anchorProof: AnchorProof): Promise<void> {
-        const decoded = decode(anchorProof.txHash.multihash);
+        const decoded = decode(Buffer.from(anchorProof.txHash.bytes));
```

Comment:
This doesn't seem right. It used the multihash before. not the full CID bytes. Can you confirm that this returns a valid tx hash?

Basically try to go from hex ethTx -> CID -> ethTx.

Reply:
> You know that you can push changes to the dependabot PRs right? :)

yep

Reply:
> This doesn't seem right. It used the multihash before. not the full CID bytes. Can you confirm that this returns a valid tx hash?
> 
> Basically try to go from hex ethTx -> CID -> ethTx.

I'm not that familiar with changes introduced by CID v1. That's why I opened a PR in the first place. I know that you've been into the material more :)

Reply:
We should likely keep using `.mulihash`! Just convert it to a Buffer here.

Reply:
See here: https://github.com/multiformats/js-cid#cidmultihash
bytes returns the full CID.

Reply:
thanks!